### PR TITLE
Export to KML

### PR DESCRIPTION
**Pull request by jolavb**
Users can now export Trails and Hydrants to KML. In addition to adding export functionality I moved the import export tools into a dialog which is opened with the Import/Export button.

### CODE REVIEW

File: src/components/ImportExport.js [hunk 1]
```diff
@@ -96,55 +104,122 @@ class ImportExport extends React.Component {
...
-          <input onChange={this.changeFile} type='file' />
-          <Button variant="raised" onClick={this.importFile} > Import </Button>
-          <Button variant="raised"  onClick={this.exportFile} > Export </Button>
-        </div>
+      <div>
+          <div style={style}>
+            <Button variant="raised" onClick={this.handleOpen}>Import / Export</Button>
+          </div>
+          <Dialog onBackdropClick={this.handleClose} aria-labelledby="simple-dialog-title" open={dialogueMain} >
+            <DialogTitle id="simple-dialog-title">Import/Export</DialogTitle>
```
Comment (caacree) [inline src/components/ImportExport.js]: Drop the id and aria-labelledby, unused.

File: src/components/ImportExport.js [hunk 2]
```diff
@@ -96,55 +104,122 @@ class ImportExport extends React.Component {
...
+              </List>
+            </div>
+          </Dialog>
+
+      </div>
     )
   }
 }
 
+
```
Comment (caacree) [inline src/components/ImportExport.js]: I think one blank line is enough here.

File: src/components/ImportExport.js [hunk 3]
```diff
@@ -22,6 +28,8 @@ class ImportExport extends React.Component {
     this.state = {
       selectedFiles: null,
       exportType: null,
+      dialogueMain: false,
```
Comment (caacree) [inline src/components/ImportExport.js]: Let's name this `dialogOpen` or something a little more descriptive of it's purpose. `__Open` or `___IsOpen` is usually good for popups and dialogs. 

Also: dialogue????? Didn't realize you were so British.

**Comment by caacree** [comment]
Looks good, just little things. You can merge first if you were looking to get this up for the call, either way.

**Comment by jolavb** [comment]
Modified to set feature styles on import and export the stored features instead of  creating new ones based on values.

**Comment by caacree** [comment]
LGTM

**Pull request merged**